src/shared/ui/Tooltip.tsx

```diff
@@ -10,11 +10,14 @@ import { MarkdownView, ScrollView, YStack } from '../ui';
 
 type TooltipProps = {
   children: React.ReactNode;
-  markdown?: string;
+  markdown?: string | null;
```

Comment:
In case if B.E. will send null as empty tooltip then there is no need to have "?" ("undefined")